python/ray/train/callbacks/print.py

```diff
@@ -0,0 +1,34 @@
+import json
+from typing import Dict, List
+
+from ray.train.callbacks import TrainingCallback
+
+
+class PrintCallback(TrainingCallback):
+    """A callback that prints training results to STDOUT.
+
+    Example:
```

Comment:
Can we change the example to show usage with `Trainer`? Users shouldn't be calling `handle_result` directly, so I think this docstring could be confusing.

Reply:
Good point, we can also take the `handle_result` example and move it to the `handle_result` docstring?